.github/workflows/build.yml

```diff
@@ -21,35 +21,31 @@ on:
           - ztl
           - all
       recipe_file:
-        description: "Filename of recipe to use"
+        description: "Recipe filename to use"
         type: choice
         options:
           - recipe
           - recipe-minor
         required: true
         default: recipe
       version:
-        description: "Version of dataset being built"
+        description: "Version of dataset being built (overrides version defined in recipe)"
         type: string
         required: false
       repeat:
         description: "Repeat existing build?"
         type: boolean
         default: false
       dev_image: 
-        description: "Use dev image specific to this branch? (If exists)"
+        description: "Use dev image specific to this branch (if exists)"
         type: boolean
         required: true
         default: false
-      build_note:
-        description: "A short note about this build (optional)"
-        type: string
-        required: false
       logging_level:
         description: "Logging level"
         type: choice
         required: true
-        default: WARNING
+        default: INFO
```

Comment:
the default logging level in our python logging utils [here](https://github.com/NYCPlanning/data-engineering/blob/main/dcpy/utils/logging.py#L16) is `INFO`